### DISCUSSION TITLE
guitar pro 8 is supported (not a question, a statement.)

### DISCUSSION THREAD

**Discussion by DivannKokos** [rageagainsthepc/GuitarPro-to-Midi#21] [General]
guitar pro 8 is supported (not a question, a statement, guitar pro 8 saves to same files that guitar pro 7 saves.)

**Comment by rageagainsthepc** [comment]
Did you try that out already? There might be subtle internal changes to the format.

**Comment by DivannKokos** [comment]
nested tuplets not converting correctly (they are converted like if i loaded gp 8 file with nested tuplets in gp 7). beside that i only tested saving in gp in guitar pro 8, and converting to midi.
so simple switching to guitar pro 8 will not break anything.

**Comment by DivannKokos** [comment]
decimal tempo (50,5) not working either, not that i care about such thing (who actually uses decimal tempo...)

**Comment by DivannKokos** [comment]
![image](https://user-images.githubusercontent.com/35037670/175941538-c10a5022-27a2-42fb-977b-e81f8d01b6eb.png)
well file version inside GPs of both 7 and 8, both tell like it is 7 version

**Comment by DivannKokos** [comment]
![image](https://user-images.githubusercontent.com/35037670/175941950-66b5a266-d22f-4829-9343-76f7e7287f2e.png)
![image](https://user-images.githubusercontent.com/35037670/175942044-9911bad0-b1e8-451e-8929-bd72516817ec.png)
right file saved with gp7, left file saved with gp8
file Preferences.json from gp8 file:
`{"tonality":{"isActive":false,"value":0},"tempo":{"isActive":false,"mode":"Constant","constantTempo":100,"constantBPM":100,"fromTempo":70,"toTempo":100,"step":10,"repeatCount":1},"view":{"globalViewHeight":-1,"globalViewCurrentTrackFocusFactor":0.5,"backingTrackVisible":false,"backingTrackHeight":240}}`
file Preferences.json from gp7 file:
`{"tonality":{"isActive":false,"value":0},"tempo":{"isActive":false,"mode":"Constant","constantTempo":100,"fromTempo":70,"toTempo":100,"step":10,"repeatCount":1},"view":{"globalViewHeight":-1}}`

**Comment by DivannKokos** [comment]
"globalViewCurrentTrackFocusFactor":0.5 is about feature in gp8 focus/unfocus (amplify/attenuate) of track

backingTrackVisible":false,"backingTrackHeight":240 seems like this is about feature in gp8 adding a audio track
